### PR TITLE
Fix misuses of \grammarterm for other purposes

### PR DESCRIPTION
**Pull request by jensmaurer**
Partially addresses #1543.

### CODE REVIEW

**Comment by jensmaurer** [comment]
This is the easy part. Replacing "class-name-or-namespace-name" in [basic.lookup] requires a rephrasing (separate pull request to facility review by @zygoloid).

**Comment by tkoeppe** [comment]
Is this done? I saw you've been updating the branch.

**Comment by tkoeppe** [comment]
(Assuming it's done.)

File: source/basic.tex [hunk 1]
```diff
@@ -2240,9 +2240,9 @@
 If the \grammarterm{id-expression} in a class member access is a
 \grammarterm{qualified-id} of the form
 \begin{codeblock}
-class-name-or-namespace-name::...
+@\placeholder{class-name-or-namespace-name}@::...
 \end{codeblock}
-the \grammarterm{class-name-or-namespace-name} following the \tcode{.} or
+the \placeholder{class-name-or-namespace-name} following the \tcode{.} or
```
Comment (tkoeppe) [inline source/basic.tex]: @jensmaurer: Wait, did you mean this placeholder to also be printed in (italic) code font?

Comment (tkoeppe) [reply source/basic.tex]: Fixed.

**Pull request merged**